### PR TITLE
Add security checks

### PR DESCRIPTION
**Pull request by wistefan**
Add a nightly action to extract all container images from the submodules(based on config.json and github-releases) and run a clair-scan on them.

### CODE REVIEW

File: .github/workflows/container-scan.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: "Scan containers"
+on:
+  schedule:
+    # every night at one
+    - cron:  '0 1 * * *'
```
Comment (jason-fox) [inline .github/workflows/container-scan.yml:5]: This is too frequent - the bleeding edge is only updated once a week. The releases are updated once a month

File: .github/workflows/container-scan.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  workflow_dispatch:
+
+jobs:
+  get-containers:
+    name: Get containers from submodules
+    runs-on: ubuntu-latest
+    outputs:
+      container-matrix: ${{ steps.submodules.outputs.containers }}
```
Comment (jason-fox) [inline .github/workflows/container-scan.yml:13]: Should we be getting latest or latest release here?

Comment (wistefan) [reply .github/workflows/container-scan.yml:13]: It takes all releases. The cve-database behind the scan is updated frequently, thus we can find issues in released versions once they are found.

File: .github/workflows/container-scan.yml (hunk 1 above)
Comment (wistefan) [reply .github/workflows/container-scan.yml:5]: The CVE database is updated more frequently. To find new found issues in existing images, we need to run it more frequent than the releases are created.

File: .github/workflows/container-scan.yml (hunk 2 above)
Comment (jason-fox) [reply .github/workflows/container-scan.yml:13]: I mean the GE code base - `master` is bleeding edge, `releases` is last stable release of the GE code.

Comment (wistefan) [reply .github/workflows/container-scan.yml:13]: The versions are taken from the github-releases api, therefore only tests versions that the GE maintainer officially released(pre-releases are excluded). It does not directly work on the git code base. See https://github.com/FIWARE-Ops/get-images-from-submodules#output for some version outputs.

File: .github/workflows/container-scan.yml (hunk 1 above)
Comment (jason-fox) [reply .github/workflows/container-scan.yml:5]: NTC

File: .github/workflows/container-scan.yml (hunk 2 above)
Comment (jason-fox) [reply .github/workflows/container-scan.yml:13]: NTC

**Pull request merged**